src/Compilers/CSharp/Portable/Symbols/Source/SourceUserDefinedOperatorSymbolBase.cs

```diff
@@ -239,16 +235,23 @@ protected override void MethodChecks(BindingDiagnosticBag diagnostics)
                 return;
             }
 
-            // SPEC: All types referenced in an operator declaration must be at least as accessible 
-            // SPEC: as the operator itself.
-
-            CheckEffectiveAccessibility(_lazyReturnType, _lazyParameters, diagnostics);
             CheckValueParameters(diagnostics);
             CheckOperatorSignatures(diagnostics);
         }
 
         protected abstract (TypeWithAnnotations ReturnType, ImmutableArray<ParameterSymbol> Parameters) MakeParametersAndBindReturnType(BindingDiagnosticBag diagnostics);
```

Comment:
Did you consider whether we should:
- pull this abstract member up to `SourceOrdinaryMethodOrUserDefinedOperatorSymbol` (and have its return type changed to match the signature in `SourceOrdinaryMethodSymbolBase`).
- remove the abstract member `SourceOrdinaryMethodSymbolBase.MakeParametersAndBindReturnType`
- pull usages up to `SourceOrdinaryMethodOrUserDefinedOperatorSymbol`

I don't feel it's important to make this change, but am hoping to get some insight into your process for deciding which members to extract or not by asking this.

Reply:
> Did you consider whether we should:
...

I intentionally left this method here and didn't try to unify it with method from SourceOrdinaryMethodSymbolBase because that one has to worry about things this this class doesn't have to worry about. At least at his moment I think that it doesn't have to worry about that.